pkg/catalog/catalog.go

```diff
@@ -6,18 +6,22 @@ import (
 	"github.com/weaveworks/profiles/api/v1alpha1"
 )
 
+// Catalog contains profiles which were gathered from the underlying cluster.
 type Catalog struct {
 	profiles []v1alpha1.ProfileDescription
 }
 
+// New creates a new, empty catalog.
 func New() *Catalog {
 	return &Catalog{profiles: []v1alpha1.ProfileDescription{}}
 }
 
+// Add adds p profiles to the catalog.
 func (c *Catalog) Add(p ...v1alpha1.ProfileDescription) {
 	c.profiles = append(c.profiles, p...)
 }
 
+// Search returns catalogs which contain the given `name`.
```

Comment:
This doesn't seem quite right, the return type is a slice of `ProfileDescription` ?

Reply:
Ops, right you are.